native/chat/chat.react.js

```diff
@@ -148,17 +153,37 @@ export type ChatNavigationProp<
   RouteName: $Keys<ChatParamList> = $Keys<ChatParamList>,
 > = ChatRouterNavigationProp<ScreenParamList, RouteName>;
 
+export type ChatTopTabsNavigationProp<
+  RouteName: $Keys<ChatTopTabsParamList> = $Keys<ChatTopTabsParamList>,
+> = ChatRouterNavigationProp<ScreenParamList, RouteName>;
+
 const Chat = createChatNavigator<
```

Comment:
The way this code is organized is messy. We should always try to keep relevant code together, but right now your new stuff is defined in the middle of all of the `ChatComponent` stuff. Let's move all of your new stuff to the very top of the file.